### ISSUE TITLE
per_vertex_quality_histogram doesn`t return any value

### ISSUE THREAD

**Issue by tvoesolncee** [cnr-isti-vclab/PyMeshLab#49]
I`m interested in getting the values of quality histogram. How can I get this values? Or it will be possible in the next version of pymeshlab? All the other filters work well
```
mesh_set.apply_filter('distance_from_reference_mesh', measuremesh = 1, refmesh = 0, signeddist = False)
mesh_set.apply_filter('colorize_by_vertex_quality')
mesh_set.apply_filter('vertex_color_invert')
mesh_set.apply_filter('per_vertex_quality_histogram')
```

**Comment by alemuntoni** [comment]
I am confident that this feature will be available in the next version (same as #39)

**Comment by alemuntoni** [comment]
Solved with version 0.1.9!
